spec/v3/services/build/find_spec.rb

```diff
@@ -338,4 +338,28 @@
       )
     end
   end
+
+  describe 'including log_complete' do
+    before do
+      jobs.each do |j|
+        stub_request(:get, "http://travis-logs-notset.example.com:1234/logs/#{j.id}?by=job_id&source=api").
+          with(  headers: {
+            'Accept'=>'*/*',
+            'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+            'Authorization'=>'token notset',
+            'Connection'=>'keep-alive',
+            'Keep-Alive'=>'30',
+            'User-Agent'=>'Faraday v0.15.3'
```

Comment:
This is failing on the CI with `Faraday v0.14.0`, maybe in the backport we have a different version